pkg/kubelet/kubelet.go

```diff
@@ -1841,6 +1841,8 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 			// Return an error to signal that the sync loop should back off.
 			syncErr = fmt.Errorf("pod cannot be run: %v", runnable.Message)
 		}
+		// Increase the rejection number in metric
+		metrics.AdmissionRejectionsTotal.WithLabelValues(runnable.Reason).Inc()
```

Comment:
Similar to the resize case, soft admission failure here is continually retried, which pollutes the metric. IMO we should just remove soft admission (https://github.com/kubernetes/kubernetes/issues/125393). Either way, lets not increment the metric here.